proposals/003-installer/installer-guidelines.md

```diff
@@ -0,0 +1,79 @@
+# Installer Guidelines
+
+## 0. Objectives
+
+We want people to use Haskell, which means the path to installing Haskell for first-time users
+should be short and easy, and it should yield an environment that is easy to use with the supporting
+literature. In addition, building simple programmes with a fresh installation should be as quick
+and painless as possible.
+
+These guidelines will set out the criteria that the Haskell Foundation will use in evaluating
+installers. We do not expect any individual to meet all these criteria initially but we would like
```

Comment:
What does the Haskell Foundation provide for developers of said tool. As in: what's the incentive to meet these criteria?